controllers/subscription_controller.go

```diff
@@ -123,6 +143,81 @@ func (r *SubscriptionReconciler) loadOdfConfigMapData(ctx context.Context, logge
 	return nil
 }
 
+func (r *SubscriptionReconciler) getTargetNamespaces(olmPkgRecords []*OlmPkgRecord) []string {
+
+	var namespacesMap = make(map[string]bool)
+	var namespaces []string
+
+	for i := range olmPkgRecords {
+		if !namespacesMap[olmPkgRecords[i].Namespace] {
+			namespaces = append(namespaces, olmPkgRecords[i].Namespace)
+		}
+		namespacesMap[olmPkgRecords[i].Namespace] = true
+	}
+
+	return namespaces
+}
```

Comment:
Isn't this a duplication of the `GetTargetNamespaces` from `controllers/config.go` ?